Formula/zion.rb

```diff
@@ -0,0 +1,21 @@
+class Zion < Formula
+  desc "Provides E2EE gateway to create secure communication"
+  homepage "https://nullby1e.github.io/zion/"
+  url "https://github.com/a1eaiactaest/zion-formula/releases/download/0.0.2/zion-0.0.2.tar.gz"
+  version "0.0.2"
```

Comment:
It can get this from the url

Reply:
do you mean files? the `gateway.zip` is only on onion website, and only way to get them is to connect via proxy and curl them.

Reply:
yeah that's the plan, I'll add it soon.

Reply:
added livecheck in new commit.